staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -181,6 +182,21 @@ func (c *RESTClient) Patch(pt types.PatchType) *Request {
 	return c.Verb("PATCH").SetHeader("Content-Type", string(pt))
 }
 
+// Apply begins a PATCH request with the apply patch type.
+func (c *RESTClient) Apply(force bool, fieldManager string) *Request {
+	req := c.Patch(types.ApplyPatchType)
+
+	if force {
+		req.Param("force", "true")
```

Comment:
I'm not really sure if you should set it to false if force is false. This code relies on the default being what it is.

Reply:
I didnt set it to false when force is not  set to `true`

Reply:
Right, but how do you know that someone doing `Apply(false, "manager")` doesn't really mean they want it to false? You're assuming that the default will be false.

Reply:
I'd just `req.Param("force", strconv.FormatBool(force))`

Reply:
It can be valuable to let the user rely on the default behaviour though. I thought that was the point you were bringing up and if that is the case an enum would work